packages/edit-site/src/components/sidebar-navigation-screen-details-panel/style.scss

```diff
@@ -17,10 +17,9 @@
 
 .edit-site-sidebar-navigation-details-screen-panel__label.edit-site-sidebar-navigation-details-screen-panel__label {
 	color: $gray-600;
-	width: 100px;
-	flex-shrink: 0;
 }
 
+.edit-site-sidebar-navigation-details-screen-panel__label time,
 .edit-site-sidebar-navigation-details-screen-panel__value.edit-site-sidebar-navigation-details-screen-panel__value {
```

Comment:
We can remove the `.edit-site-sidebar-navigation-details-screen-panel__value` CSS class.